CREDITS.md

```diff
@@ -28,6 +27,7 @@
 - Christoph Mallon (Tron) - Programmer, code correctness police (0.3 - 0.5)
 - Patric Stout (TrueBrain) - NoProgrammer (0.3 - 1.2), sys op (active)
 - Thijs Marinussen (Yexo) - AI Framework, General (0.6 - 1.3)
+- Leif Linse (Zuu) - AI/Game Script (since 1.2 - 1.6)
```

Comment:
The rest of the version numbers under "Inactive developers" do not have the word "since".
```suggestion
- Leif Linse (Zuu) - AI/Game Script (1.2 - 1.6)
```

Reply:
Sorry my bad, forgot to drop that part.